packages/api/src/util/customDecorators.ts

```diff
@@ -1,21 +1,40 @@
-import {Api, ApiRx} from '@cennznet/api';
-import {SubmittableExtrinsicFunction} from '@cennznet/api/types';
+// Copyright 2019 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 import {CodecArg} from '@cennznet/types/types';
+import {Call} from '@plugnet/types/interfaces';
+import {Api, ApiRx} from '..';
```

Comment:
might create circular dependency, it would be better if you import them from their own file.